bitnami/airflow/README.md

```diff
@@ -149,6 +155,26 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `metrics.image.pullPolicy`                | Image pull policy                                                                                    | `IfNotPresent`                                               |
 | `metrics.image.pullSecrets`               | Specify docker-registry secret names as an array                                                     | `[]` (does not add image pull secrets to deployed pods)      |
 | `metrics.podAnnotations`                  | Additional annotations for Metrics exporter                                                          | `{prometheus.io/scrape: "true", prometheus.io/port: "9112"}` |
+| `additionalRepos.enabled`                 | Enables the use of additional repos in the default Airflow Workers                                   | `false` 							  |
+| `additionalRepos.repos[o].name`	    | Descriptive name of a repo to sync			 					   | `nil`							  |
```

Comment:
Typo: `additionalRepos.repos[o].name` should be `additionalRepos.repos[0].name`
You used `o` instead of `0`